configs/default/CUST-RUSRACE_F4.config

```diff
@@ -0,0 +1,114 @@
+# Betaflight / STM32F405 (S405)
+
+board_name RUSRACE_F4

```

Comment:
Is there any documentation for this target available?

Reply:
https://goo.su/RusRace

Reply:
Nice, thank you.